dev/plugins/hu.elte.txtuml.export.javascript/src/hu/elte/txtuml/export/javascript/resources/res/visualize.html

```diff
@@ -0,0 +1,50 @@
+﻿<!-- saved from url=(0014)about:internet -->
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="lib/joint.css" />
+	<!-- 3rd party libraries -->
+    <script src="lib/jquery.min.js"></script>
+    <script src="lib/lodash.min.js"></script>
+    <script src="lib/backbone-min.js"></script>
+	<script src="lib/joint.js"></script>
+	
+	<!-- visualizer libraries -->
+	<script src="visualizer/utils.js"></script>
+	<script src="visualizer/fonts.js"></script>
+	<script src="visualizer/selector.js"></script>
+	<script src="visualizer/shapes.js"></script>
+	<script src="visualizer/attributeassociationview.js"></script>
+	<script src="visualizer/nodeholders.js"></script>
+	<script src="visualizer/linkholders.js"></script>
+	<script src="visualizer/visualizers.js"></script>
+	<script src="visualizer/grid.js"></script>
+
+</head>
+<body>
+	<!-- selector link holder -->
+	<table>
```

Comment:
I suggest to add a frame in order to help understand what these lists mean. For example:

<table>
  <tbody>
    <tr align="center">
      <td colspan="2"><b>Visualized diagrams</b></td>
    </tr>
    <tr align="center">
      <td><b>Class</b></td>
      <td><b>State machine</b></td>
    </tr>
    <tr>
      <td>
        <ul>
          <li>com.example.ClassDiagram</li>
        </ul>
      </td>
      <td>
        <ul>
          <li>com.example.SMDiagram1</li>
          <li>com.example.SMDiagram2</li>
          <li>com.example.SMDiagram3</li>
        </ul>
      </td>
    </tr>
  </tbody>
</table>